src/promptflow-azure/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -624,6 +629,74 @@ def mock_trace_destination_to_cloud(subscription_id: str, resource_group_name: s
         yield
 
 
+@pytest.fixture(scope="class", autouse=is_live() and is_in_ci_pipeline())
+def counting_tokens_in_live(remote_client):
```

Comment:
may add some comments on this function, and highlight that this is for internal usage